examples/kotlin/build.gradle

```diff
@@ -40,10 +53,10 @@ dependencies {
 // in order to make them all available at runtime. Also adding the manifest so WPILib
 // knows where to look for our Robot Class.
 jar {
-    from configurations.compile.collect { it.isDirectory() ? it : zipTree(it) }
-    manifest edu.wpi.first.gradlerio.GradleRIOPlugin.javaManifest(ROBOT_CLASS)
+    from { configurations.compile.collect { it.isDirectory() ? it : zipTree(it) } }
```

Comment:
Do you want the `compile` configuration or the resolved compile configuration?
Because if you want the resolved one you need to call `configurations.compile.getResolvedConfiguration()`

Reply:
This is what we want. Their docs say this, and this is how we've always done this.

Reply:
Okay.